### PR TITLE
Key Vault: added certificate changelog

### PR DESCRIPTION
**Pull request by iscai-msft**

### CODE REVIEW

**Comment by adxsdk6** [comment]
Can one of the admins verify this patch?

File: sdk/keyvault/azure-keyvault-certificates/HISTORY.md [hunk 1]
```diff
@@ -1,5 +1,39 @@
 # Release History
 
-## 4.0.0b1 (2019-06-28)
-For release notes and more information please visit
-https://aka.ms/azure-sdk-preview1-python
+## 4.0.0b3 (2019-09-10)
+Version 4.0.0b1 is the first preview of our efforts to create a user-friendly and Pythonic client library for Azure Key Vault.
```
Comment (chlowell) [inline sdk/keyvault/azure-keyvault-certificates/HISTORY.md]: ~b1~ b3

File: sdk/keyvault/azure-keyvault-certificates/HISTORY.md [hunk 2]
```diff
@@ -1,5 +1,39 @@
...
+and
+[samples](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/keyvault/azure-keyvault-certificates/samples)
+demonstrate the new API.
+
+### Breaking changes from `azure-keyvault`:
+- Packages scoped by functionality
+    - `azure-keyvault-certificates` contains a client for certificate operations
+- Client instances are scoped to vaults (an instance interacts with one vault
+only)
+- Removed `azure.core.Configuration` from the public API in preparation for a
```
Comment (chlowell) [inline sdk/keyvault/azure-keyvault-certificates/HISTORY.md]: This was a b2 -> b3 change, so you can remove it here.

File: sdk/keyvault/azure-keyvault-certificates/HISTORY.md [hunk 3]
```diff
@@ -1,5 +1,39 @@
...
+`_create_config`, and will be removed in a future release.
+- Authentication using `azure-identity` credentials
+  - see this package's
+  [documentation](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/keyvault/azure-keyvault-keys/README.md)
+  , and the
+  [Azure Identity documentation](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/identity/azure-identity/README.md)
+  for more information
+
+### New Features:
+- Added support for HTTP challenge based authentication, allowing clients to
```
Comment (chlowell) [inline sdk/keyvault/azure-keyvault-certificates/HISTORY.md]: This was an `azure-keyvault` feature, and was added in b2 (so not new here).

File: sdk/keyvault/azure-keyvault-certificates/HISTORY.md (hunk 1 above)
Comment (iscai-msft) [reply sdk/keyvault/azure-keyvault-certificates/HISTORY.md]: thanks!

File: sdk/keyvault/azure-keyvault-certificates/HISTORY.md (hunk 2 above)
Comment (iscai-msft) [reply sdk/keyvault/azure-keyvault-certificates/HISTORY.md]: got rid of config bullet, thanks!

File: sdk/keyvault/azure-keyvault-certificates/HISTORY.md (hunk 1 above)
Comment (iscai-msft) [reply sdk/keyvault/azure-keyvault-certificates/HISTORY.md]: thanks!

File: sdk/keyvault/azure-keyvault-certificates/HISTORY.md (hunk 3 above)
Comment (iscai-msft) [reply sdk/keyvault/azure-keyvault-certificates/HISTORY.md]: removed this as a new feature, thanks!

**Pull request merged**